controllers/dspipeline_params.go

```diff
@@ -513,9 +553,18 @@ func (p *DSPAParams) ExtractParams(ctx context.Context, dsp *dspa.DataSciencePip
 		setResourcesDefault(config.MlPipelineUIResourceRequirements, &p.MlPipelineUI.Resources)
 	}
 
-	// TODO (gfrasca): believe we need to set default VisualizationServer and WorkflowController Images here
+	// TODO (gfrasca): believe we need to set default WorkflowController Images here
+
+	var err error
+
+	if p.UsingV2Pipelines(dsp) {
```

Comment:
related to above, lets move this condition into the previous, more generic SetupMLMD method

Reply:
Done.